### ISSUE TITLE
Improper formatting when fixing (prettier) errors

### ISSUE THREAD

**Issue by jtsom** [vuejs/eslint-plugin-vue#842]
I think this module is the cause of this error, but I'm not sure...

**Tell us about your environment**	
* **ESLint version:**	^5.15.1
* **eslint-plugin-vue version:**	^5.2.2
* **Node version:**	11.10.0

**Please show your full configuration:**

```json
module.exports = {
  root: true,
  env: {
    node: true,
  },
  extends: ['plugin:vue/essential', '@vue/prettier'],
  rules: {
    'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    singleQuote: true,
    'prettier/prettier': [
      'warn',
      {
        singleQuote: true,
        semi: true,
        trailingComma: 'es5',
        printWidth: 140,
        jsxBracketSameLine: true,
      },
    ],
  },
  parserOptions: {
    parser: 'babel-eslint',
  },
};
```

**What did you do?**	

```vue
<template v-slot:header>Word Bank</template>
```

**What did you expect to happen?**
VS Code shows a linting error:
<img width="503" alt="image" src="https://user-images.githubusercontent.com/13488/53571819-ca9c6d00-3b37-11e9-9565-60e741499729.png">

<img width="702" alt="image" src="https://user-images.githubusercontent.com/13488/53571804-c07a6e80-3b37-11e9-8fcf-832a3751dff5.png">

Fixing the error should format as:

<img width="295" alt="image" src="https://user-images.githubusercontent.com/13488/53571874-ec95ef80-3b37-11e9-8f24-815dab24a7d8.png">

**What actually happened?**

Formats as:
<img width="308" alt="image" src="https://user-images.githubusercontent.com/13488/53571905-fd466580-3b37-11e9-9247-5016a8af9ed4.png">

**Comment by ota-meshi** [comment]
Thank you for this issue.

I think that it is the behavior of `eslint-plugin-prettier`.
Please post the issue to the `eslint-plugin-prettier` team.

**Comment by mysticatea** [comment]
Closing as this is not a problem in this repo.

I recommend disabling ESLint rules about format when you use a code formatter.

**Comment by kaaposc** [comment]
> I recommend disabling ESLint rules about format when you use a code formatter.

If I may bug you a little - is it possible to somehow extract `<!-- eslint-disable prettier/prettier -->` from each `<template>` to the eslint configuration settings?
